src/locales/it/translation.json

```
@@ -462,154 +462,154 @@
+                "title": "Posizione dei Mostri nel contenitore di The Witcher",
                 "subtitle": "$t(communityLinks.contentType.member)",
-                "desc": "A detailed guide on locations of all the monster minis in their respective trays (Big Box only).",
+                "desc": "Una guida dettagliata sulla posizione di tutte le miniature dei mostri nei rispettivi contenitori (solo Big Box)",
                 "link": "https://boardgamegeek.com/filepage/259976/witcher-monster-tray-position"
             },
             {
-                "title": "Card and Token Organizers",
+                "title": "Organizzatori di carte e segnalini",
                 "subtitle": "$t(communityLinks.contentType.member)",
-                "desc": "3D printer files for card organizers to better fit in the Big Box, as well as token organizers.",
+                "desc": "Files per stampante 3D di organizzatori di carte e segnalini che si adattano meglio alla Big Box.",
                 "link": "https://thingiverse.com/thing:6078120"
             }
         ]
     },
     "lostMount": {
-        "title": "Lost Mount Locations"
+        "title": "Luoghi del Destriero Perduto"
     },
     "footer": {
-        "aboutTitle": "About me",
+        "aboutTitle": "Su di me",
         "aboutBody": [
-            "I am a one-man team and I work on this unofficial fan website in my spare time. If you appreciate the work I do, consider buying me a coffee!",
-            "Updates to this site will continuously roll out. If you're interested in participating in the improvement of this tool, please check out the GitHub links to fill out bug reports, enhancement suggestions, see my current tasks and ideas, participate in polls for the next major improvement/feature I work on, and more!"
+            "Sono un 'one-man team' e lavoro su questo \"fan site\" non ufficiale nel mio tempo libero. Se apprezzi il lavoro che faccio, valuta l'idea di offrirmi un caffè!",
+            "Il sito verrà aggiornato costantemente. Se sei interessato a partecipare al miglioramento di questo tool, usa i collegamenti a GitHub per compilare segnalazioni di bug, suggerimenti di miglioramento, vedere le mie attività e idee attuali, partecipare a sondaggi per i prossimi importanti miglioramenti/funzionalità su cui lavorare e altro ancora!"
         ],
         "github": "GitHub",
-        "projectTimeline": "Project Timeline",
+        "projectTimeline": "Cronologia del progetto",
         "githubLinks": [
             {
-                "text": "Find a bug?",
-                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
+                "text": "Hai trovato un bug?",
+                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool-Issues/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
```

can you update this to the previous version? I have updated this url to consolidate to the primary repo now that it is public.

```suggestion
                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
```